src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -160,6 +160,11 @@
             "SuppressWarningsHolder"
     ));
 
+    private static final Set<String> INTERNAL_MODULES = Collections.unmodifiableSet(
+            new HashSet<>(Collections.singletonList(
```

Comment:
same here

By the way, this is duplicated several times, may be extract this list to some common class?

Reply:
Done, moved to `Definitions`.
Since I moved `INTERNAL_MODULES` to `Definitions` class, it needs unmodifiable object.